### PR TITLE
First pass of changes

### PR DESCRIPTION
**Pull request by gingerbeardman**

### CODE REVIEW

File: Translations/Web/lang_en.php [hunk 1]
```diff
@@ -24,37 +24,37 @@
...
 
 // Info
 $content_info_title1 = "So simple, so strong";
 $content_info_title2 = "Privacy is very important";
 $content_info_title3 = "Still too big...";
-$content_info_text1 = "You do not even need to open Keka to compress a file, keep it in your dock and use it from there.<br />Just drag and drop your files and folders over it or over the Keka window to create a slimmer version of them.";
+$content_info_text1 = "You do not even need to open Keka to compress a file, keep it in your dock and use it from there.<br />Just drag and drop your files and folders to the Dock icon or the Keka window to create a slimmer version of them.";
 $content_info_text2 = "Share safely just setting a password and creating highly encrypted files.<br />Using AES-256 encryption specification for your 7z files and<br />Zip 2.0 legacy encryption specification for your Zip files.";
-$content_info_text3 = "If the files are inmensely huge and they do not fit your mail or server, just split them into pieces.<br />Do not worry, they will rejoin to be your original file again :)";
-$content_info_compression = "Keka can create files using this formats:";
```
Comment (aonez) [inline Translations/Web/lang_en.php]: I feel stupid, this -> these 😰

File: Translations/Web/lang_en.php [hunk 2]
```diff
@@ -24,37 +24,37 @@
...
-$content_info_extraction = "And extract all this formats:";
+$content_info_text3 = "If the files are really huge and they do not fit your mail or server, just split them into pieces.<br />Don't worry, they will still expand to produce your original file :)";
+$content_info_compression = "Keka can create files of these types:";
```
Comment (aonez) [inline Translations/Web/lang_en.php]: Maybe "of these format types"? I really like there to say format 😂

File: Translations/Web/lang_en.php [hunk 3]
```diff
@@ -24,37 +24,37 @@
...
+$content_beta_text = "You can test the newest features of Keka before they are released.<br />If you find a bug or have something to report, head over to ";
+$content_beta_unavailable = "There is no beta available at the moment.";
 $content_beta_up = "Get the latest version";
 	
 
 // Legacy
 $content_plataform_legacy =  "For $version_replace";
 $content_legacy_title = "Legacy versions of Keka";
-$content_legacy_text = 'Over the years your Mac will become older and will not support<br />the newest Keka, but relax <i class="fa fa-coffee" aria-hidden="true"></i>, those will be with you.';
+$content_legacy_text = 'Over the years your Mac will become older and will not support<br />the newest Keka, but relax <i class="fa fa-coffee" aria-hidden="true"></i>, the old versions will are here.';
```
Comment (aonez) [inline Translations/Web/lang_en.php]: You meant "will be here"? I like your phrase better.

**Comment by gingerbeardman** [comment]
Sorry not sure I reviewed the changes correctly.

**Pull request merged**